addons/account/models/account_partial_reconcile.py

```diff
@@ -121,6 +121,10 @@ def unlink(self):
         # Remove the matching numbers before reversing the moves to avoid trying to remove the full twice.
         full_to_unlink.unlink()
 
+        for line in all_reconciled:
+            if line.move_id.origin_payment_id:
+                line.move_id.origin_payment_id.invoice_ids = None
```

Comment:
don't think it will change anything but i think we usually set it to False instead of None